engine/testutil/nodes.go

```diff
@@ -626,201 +617,6 @@ func WithGenericNode(genericNode *testmock.GenericNode) VerificationOpt {
-		err = mempoolCollector.Register(metrics.ResourceDiscardedResultID, node.DiscardedResultIDs.Size)
-		require.Nil(t, err)
-	}
-
-	if node.BlockIDsCache == nil {
-		node.BlockIDsCache, err = stdmap.NewIdentifiers(1000)
-		require.Nil(t, err)
-
-		// registers size method of backend for metrics
-		err = mempoolCollector.Register(metrics.ResourceCachedBlockID, node.BlockIDsCache.Size)
-		require.Nil(t, err)
-	}
-
-	if node.PendingReceiptIDsByBlock == nil {
-		node.PendingReceiptIDsByBlock, err = stdmap.NewIdentifierMap(receiptsLimit)
-		require.Nil(t, err)
-
-		// registers size method of backend for metrics
-		err = mempoolCollector.Register(metrics.ResourcePendingReceiptIDsByBlock, node.PendingReceiptIDsByBlock.Size)
-		require.Nil(t, err)
-	}
-
-	if node.ReceiptIDsByResult == nil {
-		node.ReceiptIDsByResult, err = stdmap.NewIdentifierMap(receiptsLimit)
-		require.Nil(t, err)
-
-		// registers size method of backend for metrics
-		err = mempoolCollector.Register(metrics.ResourceReceiptIDsByResult, node.ReceiptIDsByResult.Size)
-		require.Nil(t, err)
-	}
-
-	if node.ChunkIDsByResult == nil {
-		node.ChunkIDsByResult, err = stdmap.NewIdentifierMap(chunksLimit)
-		require.Nil(t, err)
-
-		// registers size method of backend for metrics
-		err = mempoolCollector.Register(metrics.ResourceChunkIDsByResult, node.ChunkIDsByResult.Size)
-		require.Nil(t, err)
-	}
-
-	if node.VerifierEngine == nil {
-		rt := fvm.NewInterpreterRuntime()
-
-		vm := fvm.NewVirtualMachine(rt)
-
-		blockFinder := fvm.NewBlockFinder(node.Headers)
-
-		vmCtx := fvm.NewContext(
-			node.Log,
-			fvm.WithChain(node.ChainID.Chain()),
-			fvm.WithBlocks(blockFinder),
-		)
-
-		chunkVerifier := chunks.NewChunkVerifier(vm, vmCtx)
-
-		approvalStorage := storage.NewResultApprovals(node.Metrics, node.DB)
-
-		node.VerifierEngine, err = verifier.New(node.Log,
-			collector,
-			node.Tracer,
-			node.Net,
-			node.State,
-			node.Me,
-			chunkVerifier,
-			approvalStorage)
-		require.Nil(t, err)
-	}
-
-	if node.MatchEngine == nil {
-		node.MatchEngine, err = match.New(node.Log,
-			collector,
-			node.Tracer,
-			node.Net,
-			node.Me,
-			node.PendingResults,
-			node.ChunkIDsByResult,
-			node.VerifierEngine,
-			assigner,
-			node.State,
-			node.PendingChunks,
-			node.Headers,
-			requestInterval,
-			int(failureThreshold))
-		require.Nil(t, err)
-	}
-
-	if node.FinderEngine == nil {
-		node.FinderEngine, err = finder.New(node.Log,
-			collector,
-			node.Tracer,
-			node.Net,
-			node.Me,
-			node.State,
-			node.MatchEngine,
-			node.CachedReceipts,
-			node.PendingReceipts,
-			node.ReadyReceipts,
-			node.Headers,
-			node.ProcessedResultIDs,
-			node.DiscardedResultIDs,
-			node.PendingReceiptIDsByBlock,
-			node.ReceiptIDsByResult,
-			node.BlockIDsCache,
-			processInterval)
-		require.Nil(t, err)
-	}
-
-	return node
-}
-
 // NewVerificationNode creates a verification node with all functional engines and actual modules for purpose of
 // (integration) testing.
 //
```

Comment:
Should we rename this `VerificationNode` per comment on line 623/827?

Reply:
fixed in https://github.com/onflow/flow-go/pull/831/commits/4ad4c77709101c6ea7105867a8b3090fdb20d17c